api/src/main/java/org/apache/apex/api/users/ControlTuple.java

```diff
@@ -16,15 +16,15 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.apex.api;
+package org.apache.apex.api.users;
```

Comment:
Agree, operator is a simple name and representative. Works for me. @vrozov what do you think.